doc/releases/release-notes-3.3.rst

```diff
@@ -1099,6 +1099,35 @@ Libraries / Subsystems
 
   * The ``lwm2m_senml_cbor_*`` files have been regenerated using zcbor 0.6.0.
 
+* POSIX API
+
+  * Harmonize posix type definitions across the minimal libc, newlib and picolibc.
+
+    - Abstract ``pthread_t``, ``pthread_key_t``, ``pthread_cond_t``,
```

Comment:
We use `*` for nested lists every else.

Reply:
I've seen a bunch of places in the release notes where `-` is used already.

I can make those changes though. It just occurred to me that the deprecated bits should go into the deprecated section.